docs/fundamentals/code-analysis/configuration-files.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Configuration files for code analysis rules
+description: Learn about different configuration files to configure code analysis rules.
+ms.date: 09/24/2020
+ms.topic: conceptual
+no-loc: ["EditorConfig"]
+---
+# Configuration files for code analysis rules
+
+Code analysis rules have various [configuration options](configuration-options.md). These options are specified as key-value pairs in an analyzer configuration file. Following configuration files can be used to specify these options:
+
+- [EditorConfig](#editorconfig) file: File or folder based configuration options
+- [Global AnalyzerConfig](#global-analyzerconfig) file: Project level configuration options
+
+## EditorConfig
+
+[EditorConfig](/visualstudio/ide/create-portable-custom-editor-options) files are used to provide **options that apply to specific source files or folders**. Options are placed under section headers to identify the applicable files and folders. Add an entry for each rule you want to configure, and place it under the corresponding file extension section, for example, `[*.cs]`.
+
+```ini
+[*.cs]
+<%option_name%> = <%option_value%>
```

Comment:
I think the angle brackets already signify a place holder, so don't think we need the % symbols too.

```suggestion
<option_name> = <option_value>
```